src/ck-ldb/LBManager.C

```diff
@@ -1116,10 +1116,7 @@ void LBChangePredictor(LBPredictorFunction* model)
 void LBSetPeriod(double second)
 {
 #if CMK_LBDB_ON
-  if (CkpvAccess(lbmanagerInited))
-    LBManager::Object()->SetLBPeriod(second);
-  else
-    _lb_args.lbperiod() = second;
+  _lb_args.lbperiod() = second;
```

Comment:
This still doesn't seem to call periodic LB. I believe the issue is that the code that checks lbperiod setTimer() in LBManager init() fn is executed before LBSetPeriod() is called by user code. Maybe making another setTimer() from inside LBSetPeriod() would fix the issue.

Reply:
Right, I think the behavior should be that periodic LB only happens when the user passes `+LBPeriod <period>` on the command line. In its current state, this PR has that behavior. It allows the user to change the period, but that only has any effect on LB frequency when they have passed the `+LBPeriod <period>` flag.

What do you think the behavior should be? Should the user be allowed to start periodic LB by just making a call to `SetLBPeriod` from within the program without having passed the requisite flag? If so, adding a call to `setTimer` here as you suggested should fix that. If not, then I suppose that we should maybe print out a warning when the user tries to use this when periodic LB mode isn't enabled.

Reply:
It might be helpful to enable periodic LB when LBSetPeriod/SetLBPeriod, since user wouldn't need to pass the additional command line argument. Based on discussion with Ronak, this approach might be okay as long as usesAtsync is set to false.

Reply:
I've added support for this in the latest commits on this PR.